pkg/space/space.go

```diff
@@ -46,7 +46,8 @@ func NewSpaceWithFeatureToggles(userSignup *toolchainv1alpha1.UserSignup, target
 func addFeatureToggles(space *toolchainv1alpha1.Space, toggles []toolchainconfig.FeatureToggle) {
 	var winners []string
 	for _, t := range toggles {
-		weight := int(t.Weight())
+		//the value of weight is not expected to go beyond 100 or negative , it won't overflow, hence its okay to ignore the overflow linter error
```

Comment:
you are converting from uint (which cannot be negative on it's own) to int
```suggestion
		//the value of weight is not expected to go beyond 100, it won't overflow, hence its okay to ignore the overflow linter error
```